Content.Shared/Standing/StandingStateSystem.cs

```diff
@@ -67,12 +77,22 @@ public bool IsDown(EntityUid uid, StandingStateComponent? standingState = null)
             {
                 foreach (var (key, fixture) in fixtureComponent.Fixtures)
                 {
-                    if ((fixture.CollisionMask & StandingCollisionLayer) == 0)
+                    if ((fixture.CollisionMask & StandingCollisionLayer) == 0 || fixture.Hard == false)
                         continue;
 
                     standingState.ChangedFixtures.Add(key);
                     _physics.SetCollisionMask(uid, key, fixture, fixture.CollisionMask & ~StandingCollisionLayer, manager: fixtureComponent);
                 }
+
+                // Create a temporary sensor to continue registering contacts in the disabled collision layer
+                _fixtureSystem.TryCreateFixture(
+                    uid,
+                    new PhysShapeCircle(0.1f),
+                    StandingSensorFixtureName,
+                    collisionLayer: (int) CollisionGroup.None,
+                    collisionMask: StandingCollisionLayer,
+                    hard: false,
+                    manager: fixtureComponent);
             }
```

Comment:
Why are dummy fixtures getting made.

Reply:
Similar to

https://github.com/space-wizards/space-station-14/blob/85e0a5328a28b02fc29575648cfa7421f5bfb410/Content.Shared/Climbing/Systems/ClimbSystem.cs#L305-L338

It's used for detecting when a player is no longer inside an object they were slipped / dragged into

Reply:
What's wrong with updating the existing fixture? This isn't going to have the correct bounds or ID handling.